security/try-node-to-node-encryption.md

```diff
@@ -0,0 +1,67 @@
+{: .alert .alert-danger }
+
+### Value
+
+{{ site.prodname }} supports WireGuard tunnels between nodes providing transport-level security for node-to-node traffic. WireGuard provides [formally verified](https://www.wireguard.com/formal-verification/) secure and [performant tunnels](https://www.wireguard.com/performance/) without any specialized hardware. For a deep dive in to WireGuard implementation, see [whitepaper](https://www.wireguard.com/papers/wireguard.pdf).
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **Felix configuration resource** with WireGuard configuration parameters.
+
+### Before you begin...
+
+- Verify the operating system(s) running on the nodes in the cluster [support WireGuard](https://www.wireguard.com/install/).
+
+    >**Note**: WireGuard in {{site.prodname}} does not support IPv6 at this time.
+    {: .alert .alert-info}
+- In the tech preview release, node-to-node encryption is supported on an underlying network that doesn’t require Calico to use an overlay. For example, a cluster with a routed network topology.
+
+### How to
+
+1. Install WireGuard on cluster nodes using these [instructions for your operating system](https://www.wireguard.com/install/).
+
+   >**Note**: Nodes that do not support WireGuard will not be secured by WireGuard tunnels, even if traffic running on the node to and from the pods goes to nodes that do support WireGuard.
+   {: .alert .alert-info}
+
+1. Enable WireGuard encryption across all the nodes using the following command.
+
+   ```
+   calicoctl patch felixconfiguration default --type='merge' -p '{"spec":{"wireguardEnabled":true}}'
+   ```
+
+For OpenShift, add the Felix configuration with WireGuard enabled [under custom resources]({{ site.baseurl }}/getting-started/openshift/installation#optionally-provide-additional-configuration).
+
+>**Note**: This above command can be used to change other WireGuard attributes. For a list of other WireGuard parameters and configuration evaluation, see the [Felix configuration]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition).
+   {: .alert .alert-info}
+
+To assign an IPv4 address to WireGuard interface created by Felix, you need to update the [Node resource]({{ site.baseurl }}/reference/resources/node).
+
+To disable WireGuard on a specific node with WireGuard installed, modify the host-specific Felix configuration. For example:
+```
+calicoctl patch felixconfiguration <Host-Name> --type='merge' -p '{"spec":{"wireguardEnabled":false}}'
+```
+
+#### Troubleshooting
+
+- To verify that the nodes are configured for WireGuard encryption, check the node annotation set by Felix. For example:
```

Comment:
This is only the case for kubernetes datastore, not for etcdv3.

You need to check the status field using calicoctl.  This works for both datastore types.

Reply:
Enterprise 3.0 only supports kdd, so we may need to tweak the doc for OS version.

Reply:
@bmckercher123 we can use the current text and example for Enterprise. @robbrockbank I am proposing a new example, let me know if this looks OK to you.